practice-app/website/test/event_test.py

```diff
@@ -1,26 +1,92 @@
 import unittest
-import requests
-from ..api import event
-from ...main import app
+import requests, json
+
 
 class TestEventAPI(unittest.TestCase):
 
     def setUp(self):
-        app.testing = True
-        self.app = app.test_client()
-
-    def test_events_POST(self):
-
-        # Foreign key error
-        # Adding without an error
-        # without required fields
-        # 
-        response = self.app.post('/api/v1.0/events', json={
-            'name': '', 
-            'date': '',
-            'location': '',
-            'creator_user': '',
-        })
-        json_data = response.get_json()
-        self.assertEqual(json_data.status, '200 OK')
+        self.req = "http://127.0.0.1:5000/api/v1.0/events"
+        self.headers = {'Content-type': 'application/json'}
+
+    def test_events_POST_no_user(self):
```

Comment:
Test Event API is not finalized and this version is not finished. I will be opening a pull request for it. I think it would be better if you remove the those parts since they are not related to this pull request.

Reply:
Ok, then I will add it later.

Reply:
Seems like you have added it, great job!